### PR TITLE
feat: [STO-5056]: sto burp step

### PR DESCRIPTION
**Pull request by atmollo-harness**
### Summary

Creates sto burp step 

#### Screenshots

image

Screen Shot 2023-05-12 at 2 08 39 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Comment by atmollo-harness** [comment]
retrigger coverage

File: src/modules/85-sto/components/PipelineSteps/SecurityFields.tsx [hunk 1]
```diff
@@ -423,6 +423,8 @@ interface SecurityInstanceFieldsProps extends SecurityFieldsProps<SecurityStepDa
     protocol?: boolean
     port?: boolean
     path?: boolean
+    username?: boolean
+    password?: boolean
```
Comment (chris-harness) [inline src/modules/85-sto/components/PipelineSteps/SecurityFields.tsx:427]: Don't we already have auth fields that can be reused?

Comment (atmollo-harness) [reply src/modules/85-sto/components/PipelineSteps/SecurityFields.tsx:427]: These are specific to an instance so like username and password of a site that is being scanned

**Review by williamwissemann — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**